flight/Modules/Attitude/attitude.c

```diff
@@ -952,12 +952,6 @@ static int32_t setNavigationRaw()
 //! Set the navigation information to the raw estimates
 static int32_t setNavigationNone()
 {
-	UAVObjEvent ev;
-
-	// Throw away data to prevent queue overflows
-	PIOS_Queue_Receive(gpsQueue, &ev, 0);
```

Comment:
No reason to do this; can always drain the queue when we are going to actually use it.